package.json

```diff
@@ -32,7 +32,7 @@
     "bootstrap-no-build-no-concurrency": "lerna bootstrap --no-ci --force-local --concurrency 1",
     "bootstrap-no-build-quiet": "lerna bootstrap --no-ci --force-local --loglevel=error",
     "ci:bootstrap": "lerna bootstrap --no-ci --force-local --ignore-scripts && lerna run prepublish",
-    "ci:bootstrap-no-scripts": "lerna bootstrap --no-ci --force-local --ignore-scripts",
+    "ci:bootstrap-no-scripts": "lerna bootstrap --no-ci --force-local --ignore-scripts --loglevel=info",
```

Comment:
Do we want to keep this? Guessing it's lighter logging than something like `debug` and maybe it was already implicitly set to `info`.

Reply:
Fair - it's not a needed change.   

beda507fca